app/renderer/js/utils/request-util.ts

```diff
@@ -49,7 +49,7 @@ class RequestUtil {
 		};
 	}
 
-	formatUrl(domain) {
+	formatUrl(domain: string) {
 		const hasPrefix = (domain.indexOf('http') === 0);
```

Comment:
The linter was complaining about using `startsWith` here so made that change here